src/AbstractFormHandler.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Hostnet\Component\Form;
+
+use Symfony\Component\Form\FormInterface;
+
+/**
+ * @author Yannick de Lange <ydelange@hostnet.nl>
+ */
+abstract class AbstractFormHandler implements NamedFormHandlerInterface
```

Comment:
Right now this will force people to implement the `AbstractFormHandler`, because the `FormHandlerInterface` is deprecated. It will also enforce us to hint against the implementation and not interface.